nano/node/network.cpp

```diff
@@ -215,15 +215,21 @@ bool nano::network::send_votes_cache (std::shared_ptr<nano::transport::channel>
 	return result;
 }
 
-void nano::network::flood_message (nano::message const & message_a)
+void nano::network::flood_message (nano::message const & message_a, bool const & is_dropable)
 {
 	auto list (list_fanout ());
 	for (auto i (list.begin ()), n (list.end ()); i != n; ++i)
 	{
-		(*i)->send (message_a);
+		(*i)->send (message_a, nullptr, is_dropable);
 	}
 }
 
+void nano::network::flood_block (std::shared_ptr<nano::block> block_a, bool const & is_dropable)
```

Comment:
Probably still can be in network.hpp

Reply:
was moved when I added something to the function, good catch, moved it back